test_pool/pcie/operating_system/test_os_p004.c

```diff
@@ -187,9 +187,9 @@ payload(void)
          * Read the same
          */
 
-        val_pcie_bar_mem_read(bdf, mem_base + mem_offset, &old_value);
-        val_pcie_bar_mem_write(bdf, mem_base + mem_offset, KNOWN_DATA);
-        val_pcie_bar_mem_read(bdf, mem_base + mem_offset, &read_value);
+        old_value = val_mmio_read(mem_base + mem_offset);
+        val_mmio_write(mem_base + mem_offset, KNOWN_DATA);
+        read_value = val_mmio_read(mem_base + mem_offset);
```

Comment:
Please revert back these changes as this information will be taken from platform

Reply:
Agree.